dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -480,7 +505,8 @@ def parse_query_result(self, res_lines):
         if len(rows) > 0:
             try:
                 schema, rows = self.extract_datatypes(schema, rows)
-            except Exception:
+                schema, rows = self.convert_rows(schema, rows)
```

Comment:
what is the need for extract_datatypes now  that we are using convert_rows

Reply:
we still need to identify the types, that information is not available as we have plain text to parse without metadata.